adapters/repos/db/vector/hnsw/condensor_integration_test.go

```diff
@@ -487,6 +488,68 @@ func TestCondensorWithoutEntrypoint(t *testing.T) {
 	})
 }
 
+func TestCondensorWithPQInformation(t *testing.T) {
+	rand.Seed(time.Now().UnixNano())
+	rootPath := t.TempDir()
+	ctx := context.Background()
+
+	logger, _ := test.NewNullLogger()
+	uncondensed, err := NewCommitLogger(rootPath, "uncondensed", logger,
+		WithCommitlogCycleTicker(cyclemanager.NewNoopTicker))
+	require.Nil(t, err)
+	defer uncondensed.Shutdown(ctx)
+
+	t.Run("add pq info", func(t *testing.T) {
+		uncondensed.AddPQ(ssdhelpers.PQData{
+			Ks:                  3,
+			M:                   0,
```

Comment:
Maybe safer here to set `M` to something non-zero? It should always be a positive number that divides the dimensions.

Reply:
Ah reading your reasoning in #3050